specs/007-plan-structuring/plan.md

```
@@ -0,0 +1,2711 @@
+# All scheduling unit tests
+pnpm vitest run tests/unit/scheduling
+
+# All scheduling integration tests
+pnpm vitest run tests/integration/scheduling
+
+# Specific test file
+pnpm vitest run tests/unit/scheduling/hash.spec.ts
+```
+````
+
+````
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)